website/docs/how-to/how-to-add-sso-azure-saml.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to add SSO with SAML 2.0 Azure
+---
+
+:::info Availability
+
+The **Single-Sign-On capability** is only available for customers on the Enterprise subscription. Check out the [Unleash plans](https://www.getunleash.io/plans) for details.
+
+:::
+
+## Introduction {#introduction}
+
+This guides shows you how to use [Unleash's Single-Sign-On (SSO) integration](../reference/sso.md) with SAML 2.0 and how to connect it to Azure Active Directory as an ID provider (IdP).
+
+## Basic configuration
+
+### Prequisites
+
+This guide expects you to already have:
+
+- Administrator access to the Unleash instance you want to configure
+- Azure AD access for your Azure instance 
+
+### Step 1: Create an Enterprise Application within Azure AD {#step-1}
+
+**a) Sign in to your Azure AD and **create a new Enterprise Application**. 
+
+![In the Azure directory overview, use the add button and select the enterprise application option.](/img/sso-azure-saml-add-enterprise-app.png)
```

Comment:
This is good, but I think the alt text here is useful for everyone. Maybe use a figure with a caption? 

```suggestion
<Figure caption="In the Azure directory overview, use the add button and select the enterprise application option." img="/img/sso-azure-saml-add-enterprise-app.png"/>
```